src/openrct2/ride/Vehicle.h

```diff
@@ -150,6 +150,28 @@ enum VEHICLE_STATUS
 
 struct rct_vehicle_sound_params;
 
+enum VEHICLE_TRACK_SUBPOSITION : uint8_t
+{
+    VEHICLE_TRACK_SUBPOSITION_0,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_1,
```

Comment:
1 represents the going out (left side) cable, 2 represents the return (right side) cable, 3 is the end or far bullwheel, 4 is the start or close bullwheel. We can probably make some better names based off that knowledge.